ci_tests/OpenSuse42.1/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM opensuse/archive:42.1
+
+RUN zypper --non-interactive update
```

Comment:
Do we need the update?

Reply:
I think it runs the update automatically if we don't call it explicitly.